plugins/modules/zos_job_submit.py

```diff
@@ -612,7 +613,7 @@ def query_jobs_status(module, jobId):
             )
     if not output and timeout == 0:
         raise SubmitJCLError(
-            "THE JOB CAN NOT BE QUERIED FROM JES (TIMEOUT=10s). PLEASE CHECK THE ZOS SYSTEM. IT IS SLOW TO RESPONSE."
+            "THE JOB CAN NOT BE QUERIED FROM JES (TIMEOUT=10s). PLEASE CHECK THE ZOS SYSTEM. IT IS SLOW TO RESPOND."
```

Comment:
Can we change the case to more sentence structure (upper and lower case) since we have other edits now?

Reply:
done